packages/core/src/themes/index.scss

```diff
@@ -0,0 +1,4 @@
+@forward './day';
+@forward './night';
+@forward './theme';
```

Comment:
If you agree with still having `mixins`, probably this should go there?

Reply:
To answer your question, I moved `mixins` to `utils` intentionally, as I felt they have more or less the same goal.
I felt like `helpers` are things we export out, and `utils` are internal utilities.

I always prefer folder-by-feature than folder-by-type, but I suppose `@mixin theme` would fit in `utils`, by the logic above.

I'd rather not have `mixins`, but if you feel strongly about it, how's `utils/mixins`?

Reply:
I agree that they both same/similar but the file structure is different. For example, for utils we always have unit tests - for mixins we don't. That's why the `utils` directory is now a little unorganised IMO, it's more difficult to navigate.

Maybe mixins under utilities should also go into subdirectories?

Reply:
I see. We should probably have unit tests for the mixins at some point.

I'll defer this issue for later, I just mean to refactor `core/src/core` after all.
For now I'll keep mixins.

Reply:
I think utils should also be exported, but not meant to be used as an API. For example, someone might want to use `classy`.

Reply:
Should they import from `@onfido/castor/utils` then?
Makes the official vs internal API explicit by path.

We might want to document that only the root export is the official API.

Reply:
Yes, it could be - looks nice, but previously you mentioned TypeScript issue about having imports from subdirectories?

Reply:
Can't remember what that was, but I don't foresee any issues.
Perhaps it was a matter of conflicting names?
We can revisit this problem if we encounter it in the future.